husky_description/urdf/description.xacro

```diff
@@ -31,7 +31,7 @@ ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSI
   <xacro:arg name="kinect_enabled" default="false" />
   <xacro:arg name="robotiq_grippers_enabled" default="$(optenv ROBOTIQ_GRIPPERS_ENABLED false)" />
  
-  <xacro:include filename="$(find husky_description)/urdf/husky.urdf.xacro" />
+  <xacro:include filename="$(find husky_description)/urdf/dual_arm_husky.urdf.xacro" />
```

Comment:
I've written the code inside the husky.urdf.xacro such that it is parametrized, with hopes of one day merging it into the main branch. Launch files will look for husky.urdf.xacro and so I do not want to make shotgun code.